example/lib/pages/login_page.dart

```diff
@@ -58,27 +58,39 @@ class LoginPage extends HookWidget {
                   }
                 },
               ),
-              
               ElevatedButton(
                 child: const Text('Create Random Wallet'),
                 onPressed: () async {
                   try {
                     print('Creating a random wallet...');
-                    var signer = EthPrivateKey.createRandom(Random.secure()).asSigner();
-                    print('Wallet created with address: ${await signer.address}');
+                    var signer =
+                        EthPrivateKey.createRandom(Random.secure()).asSigner();
+                    print(
+                        'Wallet created with address: ${await signer.address}');
                     print('Authorizing the wallet...');
                     bool isAuthorized = await session.authorize(signer);
                     print('Wallet authorized: $isAuthorized');
-                    print(context);
-                    print('Session: $session');
-                    print('Context: $context');
+                    if (isAuthorized) {
+                      await session.loadSaved();
+                    }
+                    // print(context);
+                    // print('Session: $session');
+                    // print('Context: $context');
+                    // Future.delayed(const Duration(seconds: 10), () {
+                    if (!context.mounted) return;
```

Comment:
I added this to avoid this warning:
https://dart.dev/tools/linter-rules/use_build_context_synchronously